deepfence_worker/cronscheduler/init_neo4j.go

```diff
@@ -28,6 +28,7 @@ func initNeo4jDatabase(ctx context.Context) error {
 	session.Run("CREATE CONSTRAINT ON (n:KubernetesCluster) ASSERT n.node_id IS UNIQUE", map[string]interface{}{})
 	session.Run("CREATE CONSTRAINT ON (n:ContainerImage) ASSERT n.node_id IS UNIQUE", map[string]interface{}{})
 	session.Run("CREATE CONSTRAINT ON (n:ImageStub) ASSERT n.node_id IS UNIQUE", map[string]interface{}{})
+	session.Run("CREATE CONSTRAINT ON (n:ImageMaskStub) ASSERT n.node_id IS UNIQUE", map[string]interface{}{})
```

Comment:
What is `ImageMaskStub`?
Should this not be `ImageTag`?

Reply:
This is the node created with the nodeid of secret/malware/vulnerability. The relationship is as follows: 
`ContainerImage -[:ALIAS] -> ImageTag - [:MASKED] -> ImageMaskStub`

Reply:
What about we do:
```
ContainerImage -[:ALIAS] -> ImageTag - [:MASKED] -> Vulnerability/Secret/Malware
```
?

Reply:
Done.